install/Knative-custom-install.md

```diff
@@ -0,0 +1,301 @@
+# Performing a Custom Knative Installation
+
+Use this guide to perform custom installations of Knative on your existing
+Kubernetes clusters. Knative's pluggable components allow you to
+install only what you want in each of your clusters.
+
+The steps covered in the guide are for advanced operators who want to
+customize each Knative installation. Installing individual Knative components
+requires that you to run multiple and separate installation commands.
+
+## Before you begin
+
+* If you are new to Knative, you should instead [install one of the default
+  installation packages](./README.md). Follow the host specific Knative installation
+  instructions to help you get up and running with the least effort. Installing
+  a default package also ensures that you can run all the
+  [Knative Serving Samples](https://github.com/knative/docs/blob/master/serving/samples/README.md).
+
+* The steps in this guide use `bash` for the MacOS or Linux environment; for Windows,
+  some commands might need adjustment.
+
+* This guide assumes that you have an existing Kubernetes cluster,
+  on which you're comfortable installing and running _alpha_ level software.
+
+* Kubernetes requirements:
+
+  * Your Kubernetes cluster version must be v1.10 or newer.
+
+  * Your version of the
+    [`kubectl` CLI tool](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    must be v1.10 or newer.
+
+## Installing Istio
+
+Knative depends on [Istio](https://istio.io/docs/concepts/what-is-istio/) for
+the service mesh.
+
+You should first install the `istio-crds.yaml` package to ensure that the Istio
+[Custom Resource Definitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+are created before installing one of the Istio install packages.
+
+### Choosing an Istio installation package
+
+There are two options for installing the Istio service mesh:
```

Comment:
I would call these two options: "Istio with a service mesh"  and "Istio without a service mesh".  The second installations will enable Knative Build/Serving but not Eventing.

Reply:
done